modules/mobile/spec/support/helpers/spec_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.before :each, type: :request do
+    Flipper.enable('va_online_scheduling')
+  end
+
+  config.define_derived_metadata(file_path: %r{modules/mobile/spec}) do |metadata|
+    metadata[:json_api] = true
+  end
+
+  config.after :each, json_api: true, type: :request do |example|
+    content_type = response.header['Content-Type']
+
+    if content_type != 'application/pdf' && response.body.present? &&
+       response.body != 'null' && !example.metadata[:skip_json_api_validation]
```

Comment:
don't validate against pdf file or nil/empty responses. Unclear if these responses are actually breaking convention or just the parser not being smart enough to not worry about nil responses. See tickets linked in the PR to evaluate this quesiton.